scripts/jenkins/Jenkinsfile

```diff
@@ -415,6 +421,52 @@ kill `pidof dbus-daemon`
     }
 }
 
+/* Generate Stages that build Debian packages
+ */
+def generate_package_build_stages() {
+    def tasks = [:]
+    tasks << build_package_debian_stretch()
+    return tasks
+}
+
+def build_package_debian_stretch() {
+    return withDockerEnv("package/stretch", DOCKER_IMAGES.stretch) {
+        withCredentials([file(credentialsId: 'jenkins-key', variable: 'KEY'),
+                         file(credentialsId: 'jenkins-secret-key', variable: 'SKEY')]) {
+            sh "gpg --import $KEY"
+            sh "gpg --import $SKEY"
+        }
+        withEnv(["DEBSIGN_PROGRAM=gpg",
+                 "DEBFULLNAME=Jenkins (User for Elektra automated build system)",
+                 "DEBEMAIL=autobuilder@libelektra.org"]) {
+            sh "rm -R ./*"
```

Comment:
sounds dangerous. Maybe put into a function "initialCleanup"?

Reply:
The function might already exist:

https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#deletedir-recursively-delete-the-current-directory-from-the-workspace

Reply:
I know I used it in the very same file. But it behaves differently as it deletes the current directory, not just the content of the current directory.